app/helpers/assessment_query_helper.rb

```diff
@@ -5,4 +5,67 @@ module AssessmentQueryHelper
   def assessments_by_patient_ids(patient_ids)
     Assessment.where(patient_id: patient_ids).order(:patient_id)
   end
+
+  def search(assessments, search)
+    return assessments if search.nil? || search.blank?
```

Comment:
`search.nil? || search.blank?` can just be `search.blank?`

Reply:
Good catch. Fixed here: https://github.com/SaraAlert/SaraAlert/pull/604/commits/30d4fbeb075c861ae8a66567d663f31a9b23ee50